cypress/support/commands.js

```diff
@@ -60,9 +60,9 @@ Cypress.Commands.add('uploadFile', (fileName, mimeType, target) => {
 			if (typeof target !== 'undefined') {
 				fileName = target
 			}
-			return cy.request('/csrftoken')
-				.then(({ body }) => body.token)
-				.then(requesttoken => {
+			cy.request('/csrftoken')
+				.then(({ body }) => {
+					const requesttoken = body.token
```

Comment:
I think the duplicate then might cause timing issues, but no idea why, so far test runs on CI seem promising